tinygrad/tensor.py

```diff
@@ -7,7 +7,7 @@
 from itertools import accumulate
 import numpy as np
 
-from tinygrad.helpers import ImageDType, argfix, make_pair, getenv, IMAGE, DEBUG, flatten, DType, dtypes, prod, all_int, round_up, merge_dicts
+from tinygrad.helpers import ImageDType, argfix, make_pair, getenv, IMAGE, DEBUG, flatten, DType, dtypes, prod, all_int, round_up, merge_dicts, fully_flatten  # noqa: E501
```

Comment:
no new # noqa: E501

Reply:
It's 7 letters too long :c
What should I do ; ;

split and add 2 extra lines like this?
![image](https://github.com/tinygrad/tinygrad/assets/135171913/c0ae9176-17b4-4c22-8581-ae42bf4fe367)

Reply:
orrrr orrrrrrr I could delete the in-between spaces! 😎

Reply:
eh don't delete spaces. put that in another line so have two `from tinygrad.helpers import ...`. i can move dtype related things into `dtype.py` later if that reads better

Reply:
like what geohot did https://github.com/tinygrad/tinygrad/blob/88ff1edcf0309ba0ce604f630c1ab0a063a66f16/tinygrad/device.py#L6